README.md

```diff
@@ -61,7 +63,7 @@ An API Usage Key will be provided to you when you register for the service, this
   "recallCampaignStartDate": "2022-01-31"
 }
 ```
-### Update a recall
+### Update a recall as Fixed
```

Comment:
Can we clear this up a little bit? Perhaps "Update a recall - set status to Fixed"

Reply:
done